src/main/java/in/divya/service/AddStudentDetails.java

```diff
@@ -0,0 +1,133 @@
+package in.divya.service;
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+
+import in.divya.validator.RegistrationValidation;
+import in.divya.model.StudentDetails;
+import in.divya.util.Validation;
+
+/**
+ * 
+ * @author divy2624
+ *
+ */
+
+public class AddStudentDetails 
```

Comment:
Classname StudentDetailService.

Add/Update/Delete word can be used for "methods"